src/AppInstallerRepositoryCore/Microsoft/Schema/Pinning_1_0/PinTable.cpp

```diff
@@ -51,14 +55,15 @@ namespace AppInstaller::Repository::Microsoft::Schema::Pinning_V1_0
         createTableBuilder.Column(ColumnBuilder(s_PinTable_PackageId_Column, Type::Text).NotNull());
         createTableBuilder.Column(ColumnBuilder(s_PinTable_SourceId_Column, Type::Text).NotNull());
         createTableBuilder.Column(ColumnBuilder(s_PinTable_Type_Column, Type::Int64).NotNull());
-        createTableBuilder.Column(ColumnBuilder(s_PinTable_Version_Column, Type::Text));
+        createTableBuilder.Column(ColumnBuilder(s_PinTable_Version_Column, Type::Text).NotNull());

```

Comment:
I think we will just fail and not get pins if there is a null, which is the same as we currently do. The only way we can get a null is if someone goes to manually edit the database. I think the only thing this would do is prevent that if their db is created with the new code.